airbyte-integrations/connectors/source-stripe/source_stripe/spec.yaml

```diff
@@ -14,14 +14,65 @@ connectionSpecification:
         Your Stripe account ID (starts with 'acct_', find yours <a
         href="https://dashboard.stripe.com/settings/account">here</a>).
       order: 0
-    client_secret:
-      type: string
-      title: Secret Key
-      description: >-
-        Stripe API key (usually starts with 'sk_live_'; find yours <a
-        href="https://dashboard.stripe.com/apikeys">here</a>).
-      airbyte_secret: true
-      order: 1
+    credentials:
```

Comment:
can you add a config migration to avoid the need to break all existing syncs?

[Here's an example](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-okta/source_okta/config_migration.py) from another connector.

The goal is to migrate existing configs by moving the client secret and client_id fields inside the credentials block

Reply:
Thanks @girarda. I just added the config_migration.

Reply:
With the config migration file, should I change the version tag from 6.0.0 to 5.6.0?